### ISSUE TITLE
kimsrv needs more disk space

### ISSUE THREAD

**Issue by josephdviviano** [TIGRLab/admin#181]
I believe it is failing as a compute node because it only has ~300 MB of free space available.

**Comment by pipitone** [comment]
I've added more space to the root partition. Sometime when we have a spare afternoon, let's rebuild kimsrv. Get rid of the workstation crap that got ansibled, mirror the root drive, put those fast 300GB drives to good use. 

Anyhow, resizing details for the curious. `/` is on `/dev/sda1` which is 15GB. `/dev/sda2` is swap and `/dev/sda3` is an old zfs cache partition. So, I extended the LVM volume thusly: 

```sh
sudo pvcreate /dev/sda3                     # makes sda3 an LVM physical volume
sudo vgextend vg_root /dev/sda3             # extends the root volume group
sudo lvresize -L +50G -r vg_root/lv_root    # resizes the root logical volume by 50GB
```
